contracts/Quest1155.sol

```diff
@@ -180,6 +181,10 @@ contract Quest1155 is ERC1155Holder, ReentrancyGuardUpgradeable, PausableUpgrade
         return questId;
     }
 
+    function referralRewardAmount() external pure returns (uint256) {
```

Comment:
I would confirm we wanna do 0 for these. Makes sense but let's confirm with product just so there are no surprises.

Reply:
Will confirm, but don't think we can give a percentage of 1155s lol